app/controllers/items/new.js

```diff
@@ -85,10 +156,22 @@ export default GoodcityController.extend({
 
   grades: Ember.computed(function() {
     return [
-      { name: "A", id: "A" },
-      { name: "B", id: "B" },
-      { name: "C", id: "C" },
-      { name: "D", id: "D" }
+      {
+        name: "A",
+        id: "A"
+      },
+      {
+        name: "B",
+        id: "B"
+      },
+      {
+        name: "C",
+        id: "C"
+      },
+      {
+        name: "D",
+        id: "D"
+      }
```

Comment:
Similarly, this is defined in `app/controllers/items/detail.js`.  Move to some other common location like GoodcityController or use mixin/service.